python/py-rfc3339-validator/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rfc3339-validator
+version             0.1.4
+revision            0
+distname            rfc3339_validator-${version}
+
+categories-append   devel
+license             BSD
+supported_archs     noarch
+platforms           {darwin any}
+
+python.versions     38 39 310 311
+python.pep517       no
+
+maintainers         nomaintainer
+
+description         A pure python RFC3339 validator
+long_description    {*}${description}
+
+homepage            https://github.com/naimetti/rfc3339-validator
+
+checksums           rmd160  fede1549109fc745688fc914624a0f6c465b9f34 \
+                    sha256  138a2abdf93304ad60530167e51d2dfb9549521a836871b88d7f4695d0022f6b \
+                    size    5513
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                        port:py${python.version}-setuptools
```

Comment:
the explicit build dependency on `setuptools` can also be removed here as it is added already by the `python` PG when `python.pep517 yes`.